### PR TITLE
WEB-670: Moving images folder and import SVGs as inline components

### PR DESCRIPTION
**Pull request by Finholt**

### CODE REVIEW

File: components/base/select/Select.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import ReactSelect, { components } from "react-select";
-import arrowSelectIcon from "../../../public/images/arrow-select-icon.svg";
-import DoubleArrowSelectIcon from "../../../public/images/double-arrows-select-icon";
-import CheckMarkIcon from "../../../public/images/check-mark-icon";
+import dropdownArrowSingle from "../../../assets/images/component-icons/dropdown-arrow-single.svg";
+import { ReactComponent as DropdownArrowDouble } from "../../../assets/images/component-icons/dropdown-arrow-double.svg";
```
Comment (Finholt) [inline components/base/select/Select.tsx:4]: This is a good example of the two ways to import SVGs now. The top is how you would import it if you wanted to use it as the src of an img tag, and the bottom is how you would import it if you wanted to use it as an inline SVG

File: components/base/select/Select.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const DefaultDropdownIndicator = (props: any) => {
 const CodeDropdownIndicator = (props: any) => {
   return (
     <components.DropdownIndicator {...props}>
-      <DoubleArrowSelectIcon />
+      <DropdownArrowDouble width={7} />
```
Comment (Finholt) [inline components/base/select/Select.tsx:30]: Note that the width (and height) props are applied to the `<svg>` tag when everything is rendered

File: custom.d.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+declare module "*.svg" {
+  import React = require("react"); // eslint-disable-line no-unused-vars
+  export const ReactComponent: React.SFC<React.SVGProps<SVGSVGElement>>;
+  const src: string;
+  export default src;
+}
```
Comment (Finholt) [inline custom.d.ts:6]: This is so that TypeScript accepts SVG files can be imported as either a src string or a react component

**Review by Finholt — COMMENTED** [review]
This looks bigger than it is; most of the changes are from moving the images folder to `/assets` and running our SVGs through [SVGOMG](https://jakearchibald.github.io/svgomg/), and then tweaking imports, etc. following that.

I also added two sub-folders to the images to categorize our icons between `content-icons` (icons that will be imported into the MDX and passed as props to the components, like those orange icons on the cards component) and `component-icons` (icons used directly within the component like arrows)

The really cool change here is using SVGR to allow us to import SVGs inline as React components (or as img src's like we have been)

File: components/base/Card.tsx [hunk 4]
```diff
@@ -96,6 +95,12 @@ const LinkStyle = styled.div`
   position: absolute;
   top: 8px;
   right: 8px;
+
+  svg {
```
Comment (ShreyaThapa) [inline components/base/Card.tsx:99]: Nice! 👍

**Review by ShreyaThapa — APPROVED** [review]
Nice touch with the comments showing examples of the new ways to import images. 😉 👍

**Pull request merged**